AzSdk.props

```diff
@@ -6,4 +6,8 @@
     <AssemblyOriginatorKeyFile>$(LibraryToolsFolder)\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   -->
+  <PropertyGroup Label="PackageMetaData">
```

Comment:
@hovsepm rather than moving this data to .net SDK repo.
The right thing would be to override these properties in your own repo.
Simply add this file in your own repo and then you don't need to change anything in the common build tools